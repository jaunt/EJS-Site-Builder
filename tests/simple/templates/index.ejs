---
generate: /
title: test 1
---

This is to test the most basic page generation: <%= title %>
