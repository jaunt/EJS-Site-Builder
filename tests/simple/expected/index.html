This is to test the most basic page generation: test 12
